Rakefile

```diff
@@ -134,7 +134,11 @@ case @osfamily
 end
 
 # The Puppet 4 load path
-@p4libdir = "/opt/puppetlabs/puppet/lib/ruby/vendor_ruby"
+@labsdir = "/opt/puppetlabs"
+@labsbindir = "#{@labsdir}/bin"
+@p4dir = "#{@labsdir}/puppet"
+@p4bindir = "#{@p4puppetdir}/bin"
```

Comment:
This one is yagni, unless we start needing there ruby. Since we still pull in system ruby for build purposes, its probably yagni for now.